Added 2 sample programs in C.

AceKnight opened the pull request:
-A Simple loop that prints number from 1 to 10.
-A Program the takes a number as an input and outputs if it's an Even or an Odd number.
-A Program the takes a number as an input and outputs if it's a Prime Number or Not.

I'm sorry if i did anything wrong. I'm new to this

jrg94 commented:
Awesome! These are all great programs, but before we can merge them, I'd like to get a formal description of the problems you're solving in the contributing doc. Could you try to take a crack at that? Otherwise, I might ask @jpvg10 to consider taking some time to do it. Then, we'll merge your programs. 

Also, since @chrboe is the resident C expert, I'll have them take a look at your code! Hope that sounds good. 😄

AceKnight commented:
> Awesome! These are all great programs, but before we can merge them, I'd like to get a formal description of the problems you're solving in the contributing doc. Could you try to take a crack at that? Otherwise, I might ask @jpvg10 to consider taking some time to do it. Then, we'll merge your programs.

 I've never done this before. I'm really sorry but I don't know how to do that 😟 .
are you talking about updating the readme file ?

AceKnight commented:
I've 
-Fixed Formatting
-Removed platform specific function: getch()
-Removed the simple loop program.
-Added description of the Programs in the Contributing document.

Please take a look and tell me if I did anything wrong. 
I've made commit to the fork. Does the Pull request update automatically?

jrg94 commented:
Hey @AceKnight! I like what you've done so far. Can you pull the latest changes from this repo? We have updated the way we handle rule descriptions, so the two sections you added today would have to go into their own files.

Let me know if I can help you through this process!

jrg94 commented:
By the way, the pull request does update automatically. However, your fork does not update automatically. You'll need to merge in the latest changes before we can merge your pull request.

AceKnight commented:
I don't know what i did just know, My CONTRIBUTING.md file is all messed up now.

jrg94 commented:
Yes! So, you basically merged all those conflicts which are denoted by the `<<<<<<<<head` lines in your file. You'll need to go through and decide which sections you want and which you don't. In other words, you'll have to remove either the section between `<<<<<head` and `======` or `==========` and `>>>>>>>>>>>>>`.

I can give more details when I have a moment.

AceKnight commented:
I hope everything is fixed now

jrg94 commented:
This is excellent! Nice work getting things working. I'll let @TheRenegadeCoder/core look over it (specifically @Noah11012 who has to approve this). Otherwise, I'm very happy with what you've accomplished.

jrg94 commented on CONTRIBUTING.md:
```diff
@@ -153,6 +153,8 @@ each link to read more about their rules.
 -   [Quine Rules][9]
 -   [Reverse a String Rules][8]
 -   [Roman Numeral Conversion][26]
```
While you're here, can you update these last three to include the word `Rules` at the end like the rest of the list?

jrg94 commented on docs/even-odd.md:
```diff
@@ -0,0 +1,22 @@
+
+## Testing
+
+Verify that the actual output matches the expected output. See the
```
I think for both this and prime you could include a table of example inputs and outputs like reverse a string or roman numeral. That way we can explore all the possible test cases. This one should include one odd number, one even number, zero, and a negative number.

jrg94 commented on docs/prime-number.md:
```diff
@@ -0,0 +1,23 @@
+
+## Testing
+
+Verify that the actual output matches the expected output. See the
```
For prime, we could include a table that has the same test cases as even/odd. One prime number, one not prime number, zero, and a negative number. It may be good to include a test case for empty input as well.

jrg94 commented on archive/c/c/even-odd.c:
```diff
@@ -0,0 +1,16 @@
+else
+    printf("%d is Odd.", a);
+
+return 0;
```
Agreed! All your code in the main block (between the braces) should be indented a level deeper.

AceKnight commented:
Hey, I fixed the indentation. Sorry, I couldn't do the Table in the docs 😟

jrg94 commented:
I don't mean to overload you! 😄 I'm just trying to offer some helpful advice. Also, we have pretty high standards for this repo because we want to provide the best possible solutions for folks who might want to see code implemented in various ways. 

That said, we really appreciate your contribution, and we hope you stick around.

AceKnight commented:
Tbh I feel really stupid 😂 . You guys have been great and I couldn't even fix this simple issue. I really hope i did everything correctly this time. 
I feel happy to contribute to this project and would like to help out in the future as well. 
I learned alot. Thanks ^^

jrg94 commented:
Don’t sweat it! A large chunk of my time is devoted to being a student, so I understand where you’re coming from! I also have 40 of my own students, so I’ve learned to be very patient for their sake 😄 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

jrg94 commented:
Welp! I said one thing and did another.. accidentally merged instead of squashing. Oh well! It's late. 😆
